src/seedu/addressbook/storage/Encryptor.java

```diff
@@ -0,0 +1,61 @@
+package seedu.addressbook.storage;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
+
+/**
+ * Encrypts and decrypts a string using AES
+ */
+public class Encryptor {
```

Comment:
Just a note, making this a "static" wrapper class may cause it to be harder to test. Perhaps making it a normal object if the behaviour of "static" class is not needed